uni/lib/view/profile/widgets/account_info_card.dart

```diff
@@ -15,9 +18,34 @@ class AccountInfoCard extends GenericCard {
 
   @override
   Widget buildCardContent(BuildContext context) {
-    return Consumer<ProfileStateProvider>(
-      builder: (context, profileStateProvider, _) {
+    return Consumer2<ProfileStateProvider, ReferenceProvider>(
+      builder: (context, profileStateProvider, referenceProvider, _) {
         final profile = profileStateProvider.profile;
+        final List<Reference> references = referenceProvider.references;
+        final Widget referenceWidgets;
+
+        if (references.isEmpty) {
+          referenceWidgets = Text(
+            "Não existem referências a pagar",
+            style: Theme.of(context).textTheme.subtitle2,
+            textScaleFactor: 0.9,
+          );
+        } else if (references.length == 1) {
+          referenceWidgets = ReferenceSection(reference: references[0]);
+        } else {
+          referenceWidgets = Column(
+              children: [
+                ReferenceSection(reference: references[0]),
+                const Divider(
+                  thickness: 1,
+                  indent: 30,
+                  endIndent: 30,
+                ),
+                ReferenceSection(reference: references[1]),
+              ]
+          );
+        }
+
```

Comment:
You can extract this logic into a function, avoiding the creation of an empty variable.